packages/data-provider/test/utils/auditUtils.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import { convertToAuditEvent, isAuditEventValid, parseAuditRecords } from '../../src/utils/auditUtils.js';
+import { AuditDataRepository, AuditDataResponse } from '../../src/utils/types.js';
+
+describe('Audit functions test', () => {
```

Comment:
`test` is redundant. maybe just `describe('Audit utils')` since we are in a file called auditUtils.spec and testing a file called `auditUtils` ;)

Reply:
renamed test description as `Audit utils`